src/components/CardBadge.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styled from "styled-components";
+
+const WrapperBadge = styled.div`
+  display: flex;
+  overflow-x: auto;
+  margin: 10px 10px;
+  flex-direction: column;
+  width: 100px;
+`;
+
+const StyledBadge = styled.span`
+  border-radius: 5px;
+  padding: 3px 10px 3px 10px;
+  font-family: "Roboto", sans-serif;
+  border: 1px solid black;
+  font-size: 20px;
+  color: #000000;
+  background: white;
+  margin-right: 5px;
+`;
+export default function CardBadge({ children }) {
+  return <StyledBadge>{children}</StyledBadge>;
+}
+
+export function FlexContainer() {
```

Comment:
Are you using `FlexContainer`? 
You already export a component in this file. If you like to create more components, better add new files for better code splitting.